nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -44,6 +44,10 @@
 
 - `samba`' `services.samba.extraConfig` setting has been removed and converted to the freeform type option named [`services.samba.settings`](#opt-services.samba.settings) according to RFC42.
 
+- `samba`' The following options of the Samba module were renamed:
+  - `enableWinbindd` -> [`winbindd.enable`](#opt-services.samba.winbindd.enable),
+  - `enableNmbd` -> [`nmbd.enable`](#opt-services.samba.nmbd.enable).
+
```

Comment:
```suggestion
```
That's not note worthy for the changelog. People will be informed by the module system already